opencti-platform/opencti-front/src/app.tsx

```diff
@@ -22,6 +23,7 @@ const App = () => (
               element={<Navigate to="/dashboard" replace={true} />}
             />
             <Route path="/dashboard/*" element={<RootPrivate />} />
+            <Route path="/public/*" element={<PublicRoot />} />
```

Comment:
it would be nicer to have the same nomenclature as with private root :
RootPrivate & RootPublic or PrivateRoot & PublicRoot

Reply:
Done